core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.html

```diff
@@ -841,10 +841,6 @@ <h3 tabindex="0">Canonical Stories</h3>
     width: fit-content;
   }
 
-  mat-icon.mat-default-theme, mat-icon {
```

Comment:
Hi @pritam2317, seems like this change the colour of _all_ mat-icons in the topic-editor-tab directive. Is that expected?

Reply:
Hello @kevintab95, the specific color `#00695e` is alreday given to `oppia-add-diagnostic-test-skills-button` selector, so this css(removed code) is not required and the mat-icons is  used only at one place i.e  here. If in future any other mat-icon is used in this page they should follow the existing pattern i.e. using css selector. 
 Thank you.

Reply:
That's good reasoning, and I agree that any future styling should use the custom selector rather than mat-icon. However, I did notice that there is a change in the colour for the "X" icon that shows up when a skill is added after the changes in the PR i.e compare the "x" before and after the changes. Is this expected and have we confirmed if there are any more **existing** mat-icon styling that need to be updated?

Reply:
@kevintab95 I have fixed the color of 'X'.

Thanks